src/Aspire.Hosting.Testing/DistributedApplicationFactory.cs

```diff
@@ -228,14 +274,14 @@ private void OnBuilderCreatingCore(DistributedApplicationOptions applicationOpti
 
     private void OnBuilderCreatedCore(DistributedApplicationBuilder applicationBuilder)
     {
+        var services = applicationBuilder.Services;
+        services.AddHttpClient();
+        services.ConfigureHttpClientDefaults(http => http.AddStandardResilienceHandler());
```

Comment:
I think @DamianEdwards turned this off by default on purpose.

Reply:
I've found the `AddStandardResilienceHandler` to cause a lot of noise in logs - #6788, I'd prefer not to see this added in a way that can't be avoided.  If your app makes heavy use of WaitFor / HealthChecks, the resiliency handler may not be needed, so the `resiliency` may translate to `unnesescary retries on failures`.

If keeping `AddStandardResilienceHandler` by default, remove it from the standard integration test template as it's no longer needed: https://github.com/dotnet/aspire/blob/e6d499928142356e9a709f6b258d13b5d8179df8/src/Aspire.ProjectTemplates/templates/aspire-xunit/9.1/IntegrationTest1.cs#L21